docs/validation_rules.md

```
@@ -85,25 +85,14 @@ Suppression:
-        isolated_ctx = ContextManager.isolate(context)
-        
-        # Modify isolated_ctx safely
-        isolated_ctx.scratchpad['result'] = data
-        
-        # Return result (executor will merge if needed)
-        return result
-    ```
-  - Alternative: If your skill doesn't need to modify context, you can ignore this warning (it's non-blocking).
-  - See: FLUJO_TEAM_GUIDE.md Section 3.5 "Idempotency in Step Policies"
+## <a id="v-ctx1"></a>V‑CTX1 — Missing required context keys
+  - Why: A step declared `input_keys` that are not produced earlier in the pipeline (including branches/imports).
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 2

(MD007, ul-indent)